src/main/java/net/mcreator/ui/blockly/BlocklyEditorToolbar.java

```diff
@@ -61,7 +62,8 @@ public class BlocklyEditorToolbar extends TransparentToolBar {
 	private JScrollablePopupMenu results = new JScrollablePopupMenu();
 	private final JButton templateLib;
 
-	private final BlocklyPanel blocklyPanel;
+	public final BlocklyPanel blocklyPanel;
+	public static BlocklyPanel staticBlocklyPanel = null;

```

Comment:
Honestly, I'm not sure if using another Blockly panel on top of the main one is a good one, because one is already a strong performance eater on smaller computers, so having to load another one may cause problems with users.